backend/pom.xml

```diff
@@ -1,54 +1,38 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
-
-	<!--
-		* Note: this pom uses 'CI friendly' versioning. For more info see:
-		* 	- https://maven.apache.org/maven-ci-friendly.html
-		*
-		* To build this project with a non-trivial version string, use the following command:
-		* 	- mvn -D revision={version} clean package
-		*
-		* To change the image name and tag, use the following command:
-		*   - mvn revision=1.2.3 -D image.name=ghcr.io/canada-dental-care-plan-backend -D image.tag=1.2.3-00000000-ALPHA.1 clean spring-boot:build-image
-		*
-		* To check for depdency updates, use the following commands:
-		* 	- mvn versions:display-property-updates
-	-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 
 	<modelVersion>4.0.0</modelVersion>
 
 
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-
-		<!-- NOTE: when updating Spring Boot, be sure to also update the <spring-boot.version> property below -->
-		<version>3.2.5</version>
+		<version>3.2.4</version>
 	</parent>
 
 
 	<groupId>ca.gov.dts-stn</groupId>
 	<artifactId>cdcp-api</artifactId>
 	<version>${revision}</version>
-	<packaging>jar</packaging>
+	<name>cdcp-api</name>
+	<description>Canadian Dental Care Plan API</description>
 
 
 	<properties>
-		<revision>0.0.0</revision>
-
 		<m2e.apt.activation>jdt_apt</m2e.apt.activation>
-		<image.name>canada-dental-care-plan-backend</image.name>
-		<image.tag>${project.version}</image.tag>
+		<image.name>cdcp-api</image.name>
 		<java.version>21</java.version>
+		<spring-boot.build-image.imageName>${image.name}</spring-boot.build-image.imageName>
+		<revision>0.0.0-00000000-0000</revision>
 
-		<immutables.version>2.10.1</immutables.version>
-		<spring-boot.version>3.2.5</spring-boot.version>
+		<h2.version>2.2.220</h2.version>
```

Comment:
Is this required? Spring Boot manages the h2 version automatically, so the only reason to override this is if we specifically need a different version.

Reply:
there is a warning message to tell the h2 version is newer than flyway can support. the h2 version was 2.2.224 selected by spring boot

Reply:
I've seen this before in the past and we usually just ignore the warning unless something is broken. It seems to work fine without pinning the version, so I think we should just let spring manage it.

The warning might go away in the next version of spring boot, but will we think to remove the h2 version then? Probably not.